.github/workflows/tests.yml

```diff
@@ -0,0 +1,19 @@
+name: tests
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
For master don't we want a cron (maybe weekly)?

Reply:
Not sure, we have to discuss yet how we will use cron jobs with our GitHub repos... but for now I would leave it out (since we didn't have that anyway until now).